themes/jamstackthemes/layouts/index.html

```diff
@@ -30,7 +30,7 @@ <h1>New Themes <span class="count">(<span class="count-number">{{ $themeCount }}
   </div>
 
   <div>
-    <div class="blurb blurb-desktop"><p>A list of themes and starters for <a href="https://jamstack.org/" target="_blank" rel="noopener">JAMstack</a> sites. We're the largest gallery of user submitted, open source themes for static site generators 🦖. This site is maintained with <a href="https://github.com/stackbithq/jamstackthemes#contribution-guidelines">community contributions</a> and with the kind support of <a href="https://www.stackbit.com">stackbit.com.</a></p></div>
+    <div class="blurb blurb-desktop"><p>A list of themes and starters for <a href="https://jamstack.org/" target="_blank" rel="noopener">JAMstack</a> sites. We're the largest gallery of user submitted, open source themes for static site generators 🦖. This site is maintained with <a href="https://github.com/stackbithq/jamstackthemes#contribution-guidelines">community contributions</a> and with the kind support of <a href="https://www.stackbit.com/&utm_source=jamstackthemes&utm_medium=referral&utm_campaign=blurb">stackbit.com.</a></p></div>
```

Comment:
`utm_campaign` should be `jamstackthemes`
`utm_content` should be `blurb`

also first param should be a `?` ie ?utm_source=

Reply:
you want both source and campaign to have the same value?

Reply:
yes, unless you have a better suggestion for the campaign value.

Reply:
we could also just remove the campaign value. its not required. and it feels like we are trying to make it fit here when its not needed.